blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/Export.java

```diff
@@ -158,6 +160,9 @@ public Object getValue(Property property, Object model, ExportConfig config) thr
                     printError(model.getClass(), e);
                     return SKIP;
                 }
+            } else if (model instanceof Item || model instanceof Run) {
```

Comment:
Its a hack but should be fine until we find something else pulling in something else during serialization thats not item or run. We can revisit if its trouble later on.